.github/workflows/builder_nodejs_slsa3.yml

```diff
@@ -127,9 +127,11 @@ jobs:
       # NOTE: The attestation name has a rng value in it's name so we don't need to check for file collisions.
       - name: Download attestations
         id: attestation-download
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: slsa-framework/slsa-github-generator/actions/nodejs/secure-attestations-download@main
         with:
```

Comment:
note: end-uses will need to use a similar command. We need to tell them that the name of the folder containing all the attestations is `${{ needs.slsa-run.outputs.attestations-download-name }}`.

@ianlewis is that OK in terms of UX?

Reply:
you said you'll provide an Action as well to simplify

Reply:
Yeah, this is fine. I have some plans for simplification.

I'd like to make the artifact uploads and the file names in the upload match if possible, and provide an action for publishing to make it easier.